### DISCUSSION TITLE
Server domain not binding to the dokploy service

### DISCUSSION THREAD

**Discussion by Vanja-S** [Dokploy/dokploy#148] [Q&A] [answered]
Hi team!

I have registered a domain with the local registrar and updated its nameservers to that of cloudflare and created an A record to point to my hetzner server as shown below:
![2024-06-18_09_16_53-Window](https://github.com/Dokploy/dokploy/assets/64472202/30ea86a0-9e42-41dc-9eff-58da57f9b4a6)

Then updated the server settings to the following:
<img width="977" alt="Screenshot 2024-06-18 at 10 11 12" src="https://github.com/Dokploy/dokploy/assets/64472202/a14c2ef2-f28e-4fa6-897e-0401e7b52e96">

However when trying to open that url in the browser I only get the "404 page not found" error? I have confirmed that the domain is registered and updated on the nameservers correctly, traefik does not yield any error logs and neither does the server.

Any ideas on what might be wrong?

I am running a fresh deployment of Dokploy, version 0.2.2 on a Ubuntu server.

**Comment by Siumauricio** [comment, marked as answer]
Hi, can you select in certificate and set as `None` and then reload traefik?

**Reply by Vanja-S** [reply]
Hey, did just that and it works! Does the certificate resolver need additional configuration?

**Reply by Siumauricio** [reply]
No that's it, there's nothing else you have to do!

**Reply by Vanja-S** [reply]
Amazing, and the same goes for deployed applications when adding domains? `None` as the certificate and probably turn off HTTPS as well?

**Reply by Siumauricio** [reply]
Correct
